SimpleInstaller/SimpleInstaller.fs

```diff
@@ -0,0 +1,86 @@
+namespace FSharp.Compiler.CodeDom
+
+// run InstallUtil.exe on this dll to register the provider in the GAC
+//
+// note: the provider build process does not sign the dll; you will need to
+// make a key with "sn -k" and add --keyfile:path_to_key to the OtherFlags
+// property in FSharp.Compiler.CodeDom.fsproj before building
+
+open System.ComponentModel
+open System.Configuration
+open System.IO
+open System.Runtime.InteropServices
+open System.Text
+open System.Xml
+
+[<RunInstaller(true)>]
+type SimpleInstaller() =
+    inherit Install.Installer()
+
+    static let machine_config_path =
```

Comment:
It looks a little odd to use "static let" though is valid - I think just make them "let"